ostelco-ios-client/Models/Context.swift

```diff
@@ -8,6 +8,35 @@
 
 // TODO: Use Region struct from Region.swift when feature/ekyc-api-integration branch is merged into develop
 
+func getRegionFromRegionResponseArray(_ regionResponses: [RegionResponse]) -> RegionResponse? {
```

Comment:
This has now become a helper function, should it be placed in a different file than the model file itself?

Reply:
@designatednerd Was using `class func` methods to do this

Reply:
@designatednerd where would you put this function?